src/amo/reducers/collections.js

```diff
@@ -315,14 +316,15 @@ type LoadCurrentCollectionPageAction = {|
 
 export const loadCurrentCollectionPage = ({
   addons,
+  numberOfAddons,
 }: LoadCurrentCollectionPageParams = {}): LoadCurrentCollectionPageAction => {
   if (!addons) {
     throw new Error('addons are required');
   }
 
```

Comment:
We should probably add a check here that `numberOfAddons` isn't `null` or `undefined`. We cannot just check against `!numberOfAddons` because `0` is a valid value, but we could check that it's a number, which would protect us against a `null` or `undefined`.

You'll notice that some of the functions in this file use `invariant` instead of an `if` statement, and I'd suggest using that here, for both the existing check for `addons` and the new one you'll add for `numberOfAddons`.

Reply:
Cool! Thanks.
I will fix it soon.